FEAT : meetup tunis first part (#68)
Fork-It-Community/forkit.community: src/content/meetup/2024-09-24-tunis.mdx

---
title: Fork it! Meetups Tunis 2024
name: Meetup Tunis, Tunisia
date: 2024-09-24
location:
  name: COMWORK
  address: 17 rue Mahmoud El Matri, Mutuelleville, 1053, Tunis
excerpt:
        For the first time in Tunisia, Fork It! is organizing a meetup!

>         The goal is to bring together  web development and UX enthusiasts for meetings focused on sharing experiences and

Reviewer: ```suggestion
        The goal is to bring together web development and UX enthusiasts for meetings focused on sharing experiences and
```

---
title: Fork it! Meetups Tunis 2024
name: Meetup Tunis, Tunisia
date: 2024-09-24
location:
  name: COMWORK
  address: 17 rue Mahmoud El Matri, Mutuelleville, 1053, Tunis
excerpt:

>         For the first time in Tunisia, Fork It! is organizing a meetup!
>         The goal is to bring together all enthusiasts of web development
>         and UX for meetings focused on sharing experiences and
>         constructive exchanges. Our community is growing and has chosen
>         Tunisia to host its first conference day in Africa. Follow us
>         and sign up for the meetup to get a taste of this unique experience!

Reviewer: ```suggestion
        For the first time in Tunisia, Fork It! is organizing a meetup!
        The goal is to bring together  web development and UX enthusiasts for meetings focused on sharing experiences and
        constructive discussions. Our community continues to grow and has chosen
        Tunisia to host its first conference day in Africa. Follow us
        and sign up for the meetup to be part of this unique experience!
```
